src/core/tests/CMakeLists.txt

```diff
@@ -567,10 +519,10 @@ endif()
 target_link_libraries(ov_core_unit_tests PRIVATE ngraph_test_util
                                         engines_test_util
                                         ngraph::builder
+                                        ngraph_reference
```

Comment:
looks like it's a dependency of `interpreter_backend`? But why is it not a `PRIVATE`?

Reply:
We use some headers from reference library